src/components/Social/Tweet.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+import colors from '../../theme/colors'
+import { mUp } from '../../theme/mediaQueries'
+import { sansSerifRegular15, sansSerifRegular18 } from '../Typography/styles'
+import { Figure } from '../Figure'
+import Image from '../Figure/Image'
+import { Header } from './Header'
+
+const styles = {
+  container: css({
+    display: 'block',
+    textDecoration: 'none',
+    borderBottom: `1px solid ${colors.text}`,
+    borderTop: `1px solid ${colors.text}`,
+    margin: '36px auto',
+    paddingTop: '10px',
+    position: 'relative',
+    maxWidth: '455px',
+    [mUp]: {
+      margin: '45px auto',
+      paddingTop: '10px'
+    }
+  }),
+  text: css({
+    ...sansSerifRegular15,
+    [mUp]: {
+      ...sansSerifRegular18
+    },
+    color: colors.text
+  })
+}
+
+const Tweet = ({
+  children,
+  attributes,
+  platform,
+  url,
+  name,
+  handle,
+  date,
+  profilePicture,
+  image
+}) => {
+  return (
+    <div {...styles.container}>
+      <Header
+        url={url}
+        profilePicture={profilePicture}
+        name={name}
+        handle={handle}
+        date={date}
+      />
+      <p {...styles.text}>{children}</p>
+      {image && (
+        <Figure>
+          <Image src={image} alt="" />
+        </Figure>
+      )}
+    </div>
+  )
+}
+
+Tweet.propTypes = {
+  children: PropTypes.node.isRequired,
+  attributes: PropTypes.object,
+  url: PropTypes.string.isRequired,
+  profilePicture: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  handle: PropTypes.string.isRequired,
+  date: PropTypes.string.isRequired,
```

Comment:
let's use a real date here (PropTypes.object.isRequired) and add `d3-time-format` as peer and a little util file:

```
import { timeFormatLocale } from 'd3-time-format'
import timeDefinition from 'd3-time-format/locale/de-CH'

export const timeFormat = timeFormatLocale(timeDefinition).format
```

Reply:
Done.